JustSaying.Sample.Restaurant.OrderingApi/Properties/launchSettings.json

```diff
@@ -0,0 +1,35 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:52289",
+      "sslPort": 44386
+    }
+  },
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "",
```

Comment:
As you've used in-process hosting, you can specify `"ancmHostingModel": "InProcess"` here to have IIS Express match.

Reply:
done